### PR TITLE
Update docs.yaml

### PR DESCRIPTION
**Pull request by emeliawilkinson24**
Added/combined the following content for DOC-4040:

  - search: 'https://(helpdocs|docs).datastax.com/(en/astra/)?(aws|gcp)/(dscloud/apollo|doc/dscloud/astra)/dscloudGettingStarted.html'
    replace: 'https://docs.datastax.com/en/astra-serverless/docs/getting-started/getting-started.html'
  - search: 'https://(helpdocs|docs).datastax.com/(en/astra/)?(aws|gcp)/(dscloud/apollo|doc/dscloud/astra)/dscloudObtainingCredentials.html'
    replace: 'https://docs.datastax.com/en/astra-serverless/docs/connect/secure-connect-bundle.html'
  - search: 'https://(helpdocs|docs).datastax.com/(en/astra/)?(aws|gcp)/(dscloud/apollo|doc/dscloud/astra)/dscloudShareClusterDetails.html'
    replace: 'https://docs.datastax.com/en/astra-serverless/docs/manage/org/manage-tokens.html'
  - search: 'http://downloads.datastax.com/java-driver/'
    replace: 'https://downloads.datastax.com/#datastax-drivers'
  - search: 'https://docs.datastax.com/en/drivers/java/(4.[0-9]+)/com/datastax/oss/driver/api/mapper/EntityHelper.html'
    replace: 'https://docs.datastax.com/en/drivers/java/\1/com/datastax/oss/driver/api/mapper/entity/EntityHelper.html'
  - search: '(http|https)://www.datastax.com/drivers/java/'
    replace: 'https://docs.datastax.com/en/drivers/java/'
  - search: 'http://docs.datastax.com/en/drivers/java'
    replace: 'https://docs.datastax.com/en/drivers/java'
  - search: 'https://docs.astra.datastax.com/docs/creating-your-astra-database'
    replace: 'https://docs.datastax.com/en/astra-serverless/docs/getting-started/create-db-choices.html'
  - search: 'https://docs.astra.datastax.com/docs/obtaining-database-credentials'
    replace: 'https://docs.datastax.com/en/astra-serverless/docs/connect/secure-connect-bundle.html'
  - search: 'https://docs.datastax.com/en/drivers/java/(4.[0-9]+)/com/datastax/oss/driver/api/core/cql/SyncCqlSession.html%60'
    replace: 'https://docs.datastax.com/en/drivers/java/\1/com/datastax/oss/driver/api/core/cql/SyncCqlSession.html'
  - search: 'https://code.google.com/p/snappy/'
    replace: 'https://google.github.io/snappy/'
  - search: 'https://code.google.com/p/guava-libraries/wiki/ListenableFutureExplained'
    replace: 'https://github.com/google/guava/wiki/ListenableFutureExplained'
  - search: 'https://community.datastax.com/index.html'
    replace: 'https://www.datastax.com/workshops'

### CODE REVIEW

File: docs.yaml [hunk 1]
```diff
@@ -37,6 +37,32 @@ rewrites:
   - http://downloads.datastax.com/php-driver/1.2.0/: http://downloads.datastax.com/php-driver/
   - http://datastax.github.io/php-driver/api/Cassandra/interface.Type/: https://docs.datastax.com/en/developer/php-driver/latest/api/Cassandra/class.Type/
   - http://datastax.github.io/php-driver/features/http://localhost:8000/features/datatypes/: https://docs.datastax.com/en/developer/php-driver/latest/features/
+  - search: 'https://(helpdocs|docs).datastax.com/(en/astra/)?(aws|gcp)/(dscloud/apollo|doc/dscloud/astra)/dscloudGettingStarted.html'
```
Comment (jgillenwater) [inline docs.yaml]: @hhughes The existing redirects in this file are formatted differently than what we added to the Java driver file. Should @emeliawilkinson24 change the redirects to match this format in this file?

cc @absurdfarce

**Comment by jgillenwater** [comment]
@hhughes @absurdfarce What do we need to change so this PR passes the checks?

**Comment by jgillenwater** [comment]
@hhughes @absurdfarce What do we need to change so this PR passes the checks?

**Comment by absurdfarce** [comment]
@jgillenwater I don't think it's a super-easy thing to fix.  The Travis build is failing on [this operation](https://github.com/datastax/php-driver/blob/master/.travis.yml#L60-L71):

```
0.21s$ if [ ! -d "${LIBUV_ROOT_DIR}" ]; then pushd /tmp; wget -q http://dist.libuv.org/dist/v${LIBUV_VERSION}/libuv-v${LIBUV_VERSION}.tar.gz; tar xzf libuv-v${LIBUV_VERSION}.tar.gz; pushd /tmp/libuv-v${LIBUV_VERSION}; sh autogen.sh; ./configure --prefix=${LIBUV_ROOT_DIR}; make -j$(nproc) install; popd; popd; else echo "Using Cached libuv v${LIBUV_VERSION}. Dependency does not need to be re-compiled"; fi
/tmp ~/build/datastax/php-driver
tar (child): libuv-v1.14.1.tar.gz: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
/home/travis/.travis/functions: line 109: pushd: /tmp/libuv-v1.14.1: No such file or directory
sh: 0: Can't open autogen.sh
/home/travis/.travis/functions: line 109: ./configure: No such file or directory
make: *** No rule to make target `install'.  Stop.
~/build/datastax/php-driver
/home/travis/.travis/functions: line 109: popd: directory stack empty
The command "if [ ! -d "${LIBUV_ROOT_DIR}" ]; then pushd /tmp; wget -q http://dist.libuv.org/dist/v${LIBUV_VERSION}/libuv-v${LIBUV_VERSION}.tar.gz; tar xzf libuv-v${LIBUV_VERSION}.tar.gz; pushd /tmp/libuv-v${LIBUV_VERSION}; sh autogen.sh; ./configure --prefix=${LIBUV_ROOT_DIR}; make -j$(nproc) install; popd; popd; else echo "Using Cached libuv v${LIBUV_VERSION}. Dependency does not need to be re-compiled"; fi" failed and exited with 1 during .
``` 

The wget command is pretty obviously not downloading anything which in turn causes everything else to fail.  I get similar results if I run locally with a setup which will cause wget to fail (for different reasons).

If I were _guessing_ about the root cause here my first thought would be a cert issue of some kind, possibly caused by some expired certs.  But that's entirely speculation.  Regardless, fixing this build is not something we can devote a lot of resources to at the moment... so if you guys have a working docs build with these changes we're prolly best just to merge them.

**Review by absurdfarce — APPROVED** [review]
Changes to doc build process only, no code or behaviour changes involved

**Pull request merged**